core/server/web/middleware/error-handler.js

```diff
@@ -107,7 +106,7 @@ _private.ThemeErrorRenderer = function ThemeErrorRenderer(err, req, res, next) {
     // It can be that something went wrong with the theme or otherwise loading handlebars
     // This ensures that no matter what res.render will work here
     // @TODO: split the error handler for assets, admin & theme to refactor this away
-    if (_.isEmpty(req.app.engines)) {
+    if (Object.keys(req.app.engines).length === 0) {
```

Comment:
Not sure if this case can come up or not but `_.isEmpty(null)` returns `true` whereas `Object.keys(null)` throws an error. I was also tempted to use this replacement in one of my changes but chose not to :-). 

@kirrg001 could you please advise on this?

Reply:
You could use

> if (req.app.engines && Object.keys(req.app.engines).length === 0)